server/src/main/java/com/broadcom/lsp/cobol/service/CobolLanguageServer.java

```diff
@@ -145,4 +187,11 @@ private ExecuteCommandOptions collectExecuteCommandList() {
     return new ExecuteCommandOptions(
         stream(ErrorCode.values()).map(ErrorCode::name).collect(toList()));
   }
+
+  /** Represents the JSON RPC response structure for shutdown command as per LSP specification */
+  @AllArgsConstructor
+  private class ShutdownResponse {
```

Comment:
Nice idea to create a class for it, but it should be either static or a separate class because it is not strongly connected with the outer class.

Reply:
done